site/en/docs/extensions/mv3/webhid/index.md

```diff
@@ -0,0 +1,61 @@
+---
+layout: "layouts/doc-post.njk"
+title: "WebHID in extensions"
+seoTitle: "Chrome Extensions: WebHID"
+date: 2023-11-02
+description: The WebHID API, which exposes Human Interface Device (HID) compatible devices to the web, is available in extensions.
+
+---
+
+A Human Interface Device (HID) takes input from or provides output to humans. It also refers to the HID protocol, a standard for bi-directional communication between a host and a device that is designed to simplify the installation procedure.
+
+This page describes aspects of the API that are particular to extensions. Refer to MDN for complete details of the [WebHID API](https://developer.mozilla.org/docs/Web/API/WebHID_API).
+
+You can find a [sample app for WebHID](https://github.com/GoogleChrome/chrome-extensions-samples/tree/main/functional-samples/sample.co2meter) in our samples repo.
+
+## Availability in extensions
+
+Chrome 117 or later.
+
+## Permissions
+
+No manifest file permissions are required; however WebHID triggers the browser's user permission flow.
+
+## Manifest
+
+No manifest keys are needed for this API.
+
+## Supporting contexts
+
+This API may be used in any extension component. Although this API cannot be used in web service workers, most of its methods and properties can be used in extension service workers. See the next section for details.
```

Comment:
I think the wording before was confusing because it said you could use the API anywhere, and then immediately said it wasn't supported in service workers.

```suggestion
This API may be used in any context. The `WebHID.requestDevice()` method cannot be used in extension service workers. See the next section for details.
```